packages/paste-core/primitives/disclosure/README.md

```diff
@@ -0,0 +1,104 @@
+# Disclosure Primitive
```

Comment:
We've fallen out of the habit of adding a README for the individual component packages. Looks like there are only a couple that have them right now. Thoughts on each package needing one.

Reply:
I tended not to care too much as the website package page should basically tell you the same information. 

Feels dangerously redundant too as they get out of sync real quick.

What would be cool long term might be the package read me _is_ the website page, but that would require a little more work to dynamically generate some common stuff on the component pages.